src/Microsoft.ML/CSharpApi.cs

```diff
@@ -3031,13 +3237,13 @@ public sealed partial class AveragedPerceptronBinaryClassifier : Microsoft.ML.Ru
             /// <summary>
             /// Init weights diameter
             /// </summary>
-            [TlcModule.SweepableFloatParamAttribute("InitWtsDiameter", 0f, 1f, numSteps:5)]
+            [TlcModule.SweepableFloatParamAttribute("InitWtsDiameter", 0f, 1f, numSteps: 5)]
             public float InitWtsDiameter { get; set; }
 
             /// <summary>
             /// Whether to shuffle for each training iteration
             /// </summary>
-            [TlcModule.SweepableDiscreteParamAttribute("Shuffle", new object[]{false, true})]
+            [TlcModule.SweepableDiscreteParamAttribute("Shuffle", new object[] { false, true })]
```

Comment:
>ct[] { fal [](start = 74, length = 10)

While we should probably change the code generator to produce more properly formatted code as we see here, I wonder if you intended to make this change? I suspect you did not, since I do not see that there was a corresponding change in `CSharpApiGenerator.cs` that would explain this format change. Perhaps something happened to this file that made your Visual Studio reformat it? I might choose to revert these changes in that file, since the next time that someone generates this file, we would revert back anyway, leading to a somewhat dirty blame.